README.md

```diff
@@ -148,7 +149,7 @@ Run a single scenario based on label name:
 To make your visual regression tests as robust as possible there are a few points to consider.
 
   - Data: Wherever you run Aye Spy you need to have complete ownership of data. Along with the ability to refresh the data back to a consistent state
-  - Dynamic elements: elements such as ads, videos, anything that moves should removed using the `removeElements` array. You want your page under test to be static.
+  - Dynamic elements: elements such as ads, videos, anything that moves should removed using the `removeElements` or `zeroOpacityElements` array. You want your page under test to be static.
```

Comment:
if we rename zeroOpacityElements to Hide 
we should explain what each one does

hide - sets the opacity of the element to 0 and will not affect the positioning of other elements on the page
remove - hard deletes the element from the Dom and may affect the positioning of other elements